src/components/MostPopular/ProductItem/ProductItem.js

```diff
@@ -86,6 +89,16 @@ class ProductItem extends Component {
     showMessage(REMOVED_FROM_WISHLIST_NOTIFICATION);
   }
 
+  handleImageError() {
+    const { errored } = this.state;
+    if (!errored) {
+      this.setState({
+        productFirstImage: 'src/assets/img/productListPage/no-image.jpeg',
```

Comment:
I would rather rename this 'no-image' to the 'default-image', just to make it more clear :)

Reply:
done)